src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -133,7 +139,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this.updateTheme(this.themeService.getColorTheme());
 		this.initActionBar();
 		this.setScrollPosition();
-		this.doLoad();
+		this.doLoad().catch((err) => console.error(err));
```

Comment:
No direct console calls - use onUnexpetedError instead

(do this for all the others in this review too)